### PR TITLE
Initial support for KML tours.

### PR DESCRIPTION
**Pull request by schmidtk**
Supports FlyTo, SoundCue, TourControl, and Wait elements for KML 2.2 w/ gx extension and KML 2.3.

I did not implement support for `AnimatedUpdate` yet. That's a much larger undertaking, and seems to be often used to control balloon visibility which we don't yet support.

Other changes:
- Updated the map container and Cesium camera `flyTo` calls to support both looking at the specified coordinate (KML `LookAt` element), and positioning the camera _at_ the specified coordinate (KML `Camera` element).
- Enabled tile loading on animation in 2D mode to avoid a black screen while tour FlyTo executes.
- Changed 3D camera FOV to 60 degrees to match Google Earth and Cesium default.
- Fixed an exception when calling `plugin.places.getPlacesRoot` to test if a KML layer node is the Saved Places node, when the KML tree was empty.

### CODE REVIEW

File: src/plugin/file/kml/tour/toursoundcue.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+  return goog.Promise.resolve();
+};
+
+
+/**
+ * Play the audio file.
+ * @private
+ */
+plugin.file.kml.tour.SoundCue.prototype.playAudio_ = function() {
```
Comment (bradh) [inline src/plugin/file/kml/tour/toursoundcue.js]: Could this just use the existing AudioManager support? Or could AudioManager be extended instead?

Comment (schmidtk) [reply src/plugin/file/kml/tour/toursoundcue.js]: The AudioManager is for shared sounds in the application and is intended to be used for notifications. We use it in external plugins to notify things like data arriving from a streaming service. KML SoundCue's typically point to music or voice overs that relate to the tour and didn't seem to fit with the AudioManager.

Comment (schmidtk) [reply src/plugin/file/kml/tour/toursoundcue.js]: I simplified SoundCue quite a bit by adding asynchronous execution support to `plugin.file.kml.tour.Wait` and extending that for the timeout behavior. Now it simply manages the audio clip and leaves the rest to Wait.

Comment (bradh) [reply src/plugin/file/kml/tour/toursoundcue.js]: OK.

Comment (wallw-bits) [reply src/plugin/file/kml/tour/toursoundcue.js]: That's fine that the sounds are not added, but I'd expect it to still obey the overall mute setting in there.

Comment (schmidtk) [reply src/plugin/file/kml/tour/toursoundcue.js]: Thanks I hadn't considered that. Fixed and tests updated.

**Pull request merged**